{"uid":"a4d19c58fb3638b","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1705626903223,"stop":1705626913439,"duration":10216},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563b7537ef83 <unknown>\n#1 0x563b75037cf7 <unknown>\n#2 0x563b7508799e <unknown>\n#3 0x563b75087aa1 <unknown>\n#4 0x563b750d2d64 <unknown>\n#5 0x563b750b10dd <unknown>\n#6 0x563b750d0006 <unknown>\n#7 0x563b750b0e53 <unknown>\n#8 0x563b75078dd4 <unknown>\n#9 0x563b7507a1de <unknown>\n#10 0x563b75343531 <unknown>\n#11 0x563b75347455 <unknown>\n#12 0x563b7532ff55 <unknown>\n#13 0x563b753480ef <unknown>\n#14 0x563b7531399f <unknown>\n#15 0x563b7536c008 <unknown>\n#16 0x563b7536c1d7 <unknown>\n#17 0x563b7537e124 <unknown>\n#18 0x7f9d0ec94ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f2de1f650d0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"c6395c3820a4dd5e46f3b075ad783b9e\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2de1e5a020>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x563b7537ef83 <unknown>\nE       #1 0x563b75037cf7 <unknown>\nE       #2 0x563b7508799e <unknown>\nE       #3 0x563b75087aa1 <unknown>\nE       #4 0x563b750d2d64 <unknown>\nE       #5 0x563b750b10dd <unknown>\nE       #6 0x563b750d0006 <unknown>\nE       #7 0x563b750b0e53 <unknown>\nE       #8 0x563b75078dd4 <unknown>\nE       #9 0x563b7507a1de <unknown>\nE       #10 0x563b75343531 <unknown>\nE       #11 0x563b75347455 <unknown>\nE       #12 0x563b7532ff55 <unknown>\nE       #13 0x563b753480ef <unknown>\nE       #14 0x563b7531399f <unknown>\nE       #15 0x563b7536c008 <unknown>\nE       #16 0x563b7536c1d7 <unknown>\nE       #17 0x563b7537e124 <unknown>\nE       #18 0x7f9d0ec94ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1705626854491,"stop":1705626854491,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1705626854491,"stop":1705626854492,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1705626854491,"stop":1705626854491,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563b7537ef83 <unknown>\n#1 0x563b75037cf7 <unknown>\n#2 0x563b7508799e <unknown>\n#3 0x563b75087aa1 <unknown>\n#4 0x563b750d2d64 <unknown>\n#5 0x563b750b10dd <unknown>\n#6 0x563b750d0006 <unknown>\n#7 0x563b750b0e53 <unknown>\n#8 0x563b75078dd4 <unknown>\n#9 0x563b7507a1de <unknown>\n#10 0x563b75343531 <unknown>\n#11 0x563b75347455 <unknown>\n#12 0x563b7532ff55 <unknown>\n#13 0x563b753480ef <unknown>\n#14 0x563b7531399f <unknown>\n#15 0x563b7536c008 <unknown>\n#16 0x563b7536c1d7 <unknown>\n#17 0x563b7537e124 <unknown>\n#18 0x7f9d0ec94ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f2de1f650d0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"c6395c3820a4dd5e46f3b075ad783b9e\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2de1e5a020>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x563b7537ef83 <unknown>\nE       #1 0x563b75037cf7 <unknown>\nE       #2 0x563b7508799e <unknown>\nE       #3 0x563b75087aa1 <unknown>\nE       #4 0x563b750d2d64 <unknown>\nE       #5 0x563b750b10dd <unknown>\nE       #6 0x563b750d0006 <unknown>\nE       #7 0x563b750b0e53 <unknown>\nE       #8 0x563b75078dd4 <unknown>\nE       #9 0x563b7507a1de <unknown>\nE       #10 0x563b75343531 <unknown>\nE       #11 0x563b75347455 <unknown>\nE       #12 0x563b7532ff55 <unknown>\nE       #13 0x563b753480ef <unknown>\nE       #14 0x563b7531399f <unknown>\nE       #15 0x563b7536c008 <unknown>\nE       #16 0x563b7536c1d7 <unknown>\nE       #17 0x563b7537e124 <unknown>\nE       #18 0x7f9d0ec94ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1705626903223,"stop":1705626913438,"duration":10215},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563b7537ef83 <unknown>\n#1 0x563b75037cf7 <unknown>\n#2 0x563b7508799e <unknown>\n#3 0x563b75087aa1 <unknown>\n#4 0x563b750d2d64 <unknown>\n#5 0x563b750b10dd <unknown>\n#6 0x563b750d0006 <unknown>\n#7 0x563b750b0e53 <unknown>\n#8 0x563b75078dd4 <unknown>\n#9 0x563b7507a1de <unknown>\n#10 0x563b75343531 <unknown>\n#11 0x563b75347455 <unknown>\n#12 0x563b7532ff55 <unknown>\n#13 0x563b753480ef <unknown>\n#14 0x563b7531399f <unknown>\n#15 0x563b7536c008 <unknown>\n#16 0x563b7536c1d7 <unknown>\n#17 0x563b7537e124 <unknown>\n#18 0x7f9d0ec94ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"de5f659de8e3dc92","name":"public ip address","source":"de5f659de8e3dc92.txt","type":"text/plain","size":12},{"uid":"99d121d2945c0133","name":"Cookies","source":"99d121d2945c0133.json","type":"application/json","size":3871},{"uid":"ed8f16e4a3e9f900","name":"Session Storage","source":"ed8f16e4a3e9f900.json","type":"application/json","size":359},{"uid":"f8edaa4ec7bd661b","name":"Local Storage","source":"f8edaa4ec7bd661b.json","type":"application/json","size":208},{"uid":"835b45a100630ffb","name":"Console Logs","source":"835b45a100630ffb.json","type":"application/json","size":2},{"uid":"b52823d71a733947","name":"Full Page Screenshot","source":"b52823d71a733947.png","type":"image/png","size":5705370},{"uid":"c0fa62252f3e97e6","name":"URL","source":"c0fa62252f3e97e6.uri","type":"text/uri-list","size":23},{"uid":"4904ebb5d9f81729","name":"Network Logs","source":"4904ebb5d9f81729.json","type":"application/json","size":23109}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1538-683"},{"name":"thread","value":"2341-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"a4d19c58fb3638b.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}